tests/core/editable/manual/enterbr.md

```diff
@@ -0,0 +1,27 @@
+@bender-tags: 4.16.3, bug, 3858
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, clipboard, enterkey, format, sourcearea
+
+
+**Note**: Do not change the cursor position during the following steps.
+
+1. Open browser's console.
+2. Copy two paragraphs placed above the editor and paste then.
+
+**Note**: This will also copy `<p>` elements that are around `a` and `b` letters.
+
+3. Hit enter twice.
+4. Type any (only one) letter and remove it using `backspace`.
+5. Don't change the cursor position. Paste again the same two paragraphs.
```

Comment:
If we want to keep the reminder here, despite the one at the beginning, maybe we can change the from a little bit:
```suggestion
5. Remember: don't change the cursor position. Paste again the same two paragraphs.
```